integration/fixtures/k8s/01-traefik-crd.yml

```diff
@@ -1356,9 +1356,20 @@ spec:
           spec:
             description: TLSStoreSpec configures a TLSStore resource.
             properties:
+              certificates:
+                items:
+                  description: Certificate holds a secret name for the TLSOption resource.
+                  properties:
+                    secretName:
+                      description: SecretName is the name of the referenced Kubernetes
+                        Secret to specify the certificate details.
+                      type: string
+                  required:
+                  - secretName
+                  type: object
+                type: array
               defaultCertificate:
-                description: DefaultCertificate holds a secret name for the TLSOption
-                  resource.
+                description: Certificate holds a secret name for the TLSOption resource.
```

Comment:
Since you made the choice to have the certificates directly within a tlsstore definition (which is the "opposite" of what we do everywhere else), this would set a precedent. I think we need to take the time to consider whether in the future (traefik3 ?) we want to abandon the notion that there could be multiple tls stores. And maybe we want to be consistent everywhere else, and also have the certificates directly in the tlsstore.

Reply:
nvm, we discussed it internally.